legal-api/report-templates/correction.html

```diff
@@ -47,8 +47,10 @@
          {% if shareClasses and (shareClassesChange or newShareClasses or ceasedShareClasses) %}
             [[common/shareStructure.html]]
          {% endif %}
-
+         
          [[common/resolutionDates.html]]
+
+         [[special-resolution/resolutionApplicationCorrection.html]]
```

Comment:
I still don't think this should be under special-resolution, this file should be under correction

Reply:
```
         [[correction/associateType.html]]
         [[correction/rulesMemorandum.html]]
         [[correction/resolution.html]]
```
what do you think? put them in separate parts? in correction.html

Reply:
Much better I think not sure if we want to include CP in the folder structure? or not.. @argush3  thoughts?

Reply:
i think that makes sense to just throw them under the `correction` folder now that we are moving to consolidate everything under the generic correction.  

And I'm siding with no CP folder.